### ISSUE TITLE
CC047 eats property trivia

### ISSUE THREAD

**Issue by ImaginaryDevelopment** [code-cracker/code-cracker#492]
[Obsolete("Only the domain should be setting this property")]
        public int AppointmentBillingStage
        {
            get { return _AppointmentBillingStage; }
            set { this.SetAndNotify(() => this.AppointmentBillingStage, ref _AppointmentBillingStage, value); }
        }

becomes

        public int AppointmentBillingStage
        {
            get { return _AppointmentBillingStage; }

            private set
            { this.SetAndNotify(() => this.AppointmentBillingStage, ref _AppointmentBillingStage, value); }
        }

**Comment by carloscds** [comment]
@ImaginaryDevelopment It's a trivia bug in set assessor, right ?

**Comment by ImaginaryDevelopment** [comment]
yeah, changing the set to private removes the attributes from the property

**Comment by carloscds** [comment]
@ImaginaryDevelopment Bug confirmed. I will fix it.
